src/c++/vdo/base/slab-depot.c

```diff
@@ -2242,13 +2242,20 @@ static void finish_reference_block_load(struct vdo_completion *completion)
 	struct pooled_vio *pooled = vio_as_pooled_vio(vio);
 	struct reference_block *block = completion->parent;
 	struct vdo_slab *slab = block->slab;
+	size_t num_blocks = vio->io_size / VDO_BLOCK_SIZE;
```

Comment:
Again, s/num_blocks/block_count/
Also could probably be a smaller type like u32 or int.